staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go

```diff
@@ -474,7 +474,7 @@ func describeLimitRangeSpec(spec corev1.LimitRangeSpec, prefix string, w PrefixW
 // DescribeLimitRanges merges a set of limit range items into a single tabular description
 func DescribeLimitRanges(limitRanges *corev1.LimitRangeList, w PrefixWriter) {
 	if len(limitRanges.Items) == 0 {
-		w.Write(LEVEL_0, "No resource limits.\n")
+		w.Write(LEVEL_0, "No LimitRange resource.\n")
```

Comment:
Nit: Should this be `No LimitRange resources...`? (plural instead of singular)

Reply:
By 'No LimitRange resource' I meant there isn't a single LimitRange resource.